BrcmPatchRAM/BlueToolFixup.cpp

```diff
@@ -154,6 +178,9 @@ static void pluginStart() {
                         shouldPatchBoardId = false;
                         break;
                     }
+            if ((getKernelVersion() == KernelVersion::Monterey && getKernelMinorVersion() >= 5) || getKernelVersion() > KernelVersion::Monterey)
+                // 12.4 Beta 3+, XNU 21.5
+                shouldPatchAddress = checkKernelArgument("-skip_address_check");
```

Comment:
This boot arg should be prefixed with `btlfx`, and is there any harm in making this default? If skipping the address check has no drawback it should just be default and unconfigurable

Reply:
`-sPREFIXED` arguments will be filtered out by macOS. Need prefix.